docs/api-docs/apps/guide/apps-01-introduction.md

```diff
@@ -1,79 +1,75 @@
-# BigCommerce Apps
+# Introduction to Building Apps
 
 <div class="otp" id="no-index">
 
 ### On This Page

```

Comment:
The section headings here and throughout the article should be in [sentence case](https://intranet.bigcommerce.com/display/SST/Headings+and+Titles)